media-sound/libopenmpt/libopenmpt-0.2.7386~beta20.3.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="OpenMPT based module player libary and command-line player"
+DESCRIPTION="libopenmpt is a cross platform C++ and C libary to decode tracked \
+music files (modules) int a raw PCM audio stream. openmpt123 is a \
+cross-platform command line or terminal based module file player."
+HOMEPAGE="https://lib.openmpt.org/libopenmpt/"
+COPYRIGHT="2004-2016 OpenMPT contributors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${portVersion/\~/-}-autotools.tar.gz"
+CHECKSUM_SHA256="a6a7e6da1ae66e1cf46985ee92c182e50652d71b96135e9fa6048e132d844753"
+SOURCE_FILENAME="libopenmpt-${portVersion/\~/-}.tar.gz"
+SOURCE_DIR="libopenmpt-0.2.7386-autotools"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libopenmpt$secondaryArchSuffix = $portVersion
+	lib:libopenmpt$secondaryArchSuffix = 0.0.20
+	cmd:openmpt123$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libmpg123$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libopenmpt${secondaryArchSuffix}_devel
+	devel:libopenmpt$secondaryArchSuffix = 0.0.20
```

Comment:
I think you should use a compat version:
```
	devel:libopenmpt$secondaryArchSuffix = 0.0.20 compat >= 0
```

Hmm, you forgot to put a REQUIRES_devel ;-)

And in BUILD_PREREQUIRES I think you should insert:
```
	cmd:aclocal
	cmd:autoconf
	cmd:autoheader
	cmd:automake
	cmd:libtoolize$secondaryArchSuffix
```
(sorted alphabeticallly with what is already in there).